website/docs/reference/project-configs/require-dbt-version.md

```diff
@@ -56,29 +74,47 @@ require-dbt-version: ">=1.0.0"
 
 </File>
 
+Remember, having an unbounded upper limit isn't recommended. Instead, check out the [pin to a range](#pin-to-a-range) example to define a range with both a lower and upper limit to ensure stability across releases.
 
 ### Pin to a range
-Use a comma separated list for an upper and lower bound. In the following example, this project will run with dbt 1.x.x.
+Use a comma separated list for an upper and lower bound. You can define a version range either as a YAML list (using square brackets) or as a comma-delimited string &mdash; both forms are valid and work.
+
+- [General range](#general-range) &mdash; Use a comma separated list to specify an upper and lower bound.
+- [Fusion-compatible range](#fusion-compatible-range) &mdash; Include 2.0.0 or greater in your version range to signal compatibility with the <Constant name="fusion_engine"/>.
+
+#### General range
+In the following examples, this project will run with dbt 1.x.x:
 
 <File name='dbt_project.yml'>
 
-```yml
-require-dbt-version: [">=1.0.0", "<2.0.0"]
-```
+```yaml
+require-dbt-version: [">=1.0.0", "<2.0.0"] # with a YAML list
+
+# OR
 
+require-dbt-version: ">=1.0.0,<2.0.0" # with a comma-delimited string
+
+```
 </File>
 
-OR
+#### Fusion-compatible range
+
+To signal compatibility with the <Constant name="fusion_engine"/>, include 2.0.0 or greater in your version range. Both YAML formats below are equivalent and valid.
```

Comment:
That last part might make me wonder why we're not showing examples that aren't valid (or if we have in the past)
```suggestion
To signal compatibility with the <Constant name="fusion_engine"/>, include 2.0.0 or greater in your version range.
```